pkg/virt-controller/services/vm.go

```diff
@@ -48,8 +48,8 @@ type VMService interface {
 	GetRunningMigrationPods(*corev1.Migration) (*v1.PodList, error)
 	CreateMigrationTargetPod(migration *corev1.Migration, vm *corev1.VM) error
 	UpdateMigration(migration *corev1.Migration) error
-	FetchVM(vmName string) (*corev1.VM, bool, error)
-	FetchMigration(migrationName string) (*corev1.Migration, bool, error)
+	FetchVM(vmName string, namespace string) (*corev1.VM, bool, error)
```

Comment:
Going to recommend that we hold off on that until a follow on patch.  I'd rather see just the straight conversion from default namespace, with the rest of the code the same land first.